pie/utils/objects.py

```diff
@@ -69,6 +84,27 @@ def _pages_from_iter(
             pages.append(embed)
         return pages
 
+    def _toggle_lock(self) -> None:
+        if self.locked:
+            self.locked = False
+            self.lock_button.label = "🔓"
+            self.lock_button.style = nextcord.ButtonStyle.green
+        else:
+            self.locked = True
+            self.lock_button.label = "🔒"
+            self.lock_button.style = nextcord.ButtonStyle.red
+
+    async def __alert_unauthorized_scroller(
+        self, interaction: nextcord.Interaction, message: str
+    ) -> None:
+        if self.ctx.guild is not None:
+            gtx = i18n.TranslationContext(self.ctx.guild.id, interaction.user.id)
+        else:
+            # TranslationContext does not know how to use user without guild,
+            # this will result in bot preference being used.
+            gtx = i18n.TranslationContext(None, interaction.user.id)
+        await interaction.response.send_message(_(gtx, message), ephemeral=True)
```

Comment:
This won't work, the second argument must be a string, not variable. You have to translate the messages where they are defined, e.g. on lines 137 + 142.

Reply:
Changed, ammended, commited